### PR TITLE
Rename jvariable to jvalue

### PR DESCRIPTION
**Pull request by musm**
As per discussion with @twadleigh

### CODE REVIEW

**Comment by musm** [comment]
Basically this function will convert a MATLAB object in the "default way" that makes the most sense to the most appropriate Julia value.

**Comment by musm** [comment]
Thoughts from others?

File: src/MATLAB.jl [hunk 1]
```diff
@@ -61,13 +61,15 @@ function nfields(mx::MxArray)
...
+function jvalue(mx::MxArray, ty::Type{AbstractString}) 
+    Base.depwarn("jvalue(mx::MxArray,ty::Type{AbstractString}) is 
+    deprecated, use jvalue(mx::MxArray,ty::Type{String}) instead. 
+    We now default to more strict typing on String types", :jvalue)
     return jstring(mx)::String
 end
 
 @deprecate duplicate(mx::MxArray) copy(mx::MxArray)
 
+@deprecate jvariable(mx::MxArray) jvalue(mx::MxArray)
```
Comment (yuyichao) [inline src/MATLAB.jl]: This doesn't look like all the deprecated signatures. (You probably don't need to specify the signatures).

File: src/MATLAB.jl [hunk 2]
```diff
@@ -61,13 +61,15 @@ function nfields(mx::MxArray)
     return mxfields(mx)
 end
 
-function jvariable(mx::MxArray, ty::Type{AbstractString}) 
-    Base.depwarn("jvariable(mx::MxArray,ty::Type{AbstractString}) is 
-    deprecated, use jvariable(mx::MxArray,ty::Type{String}) instead. 
-    We now default to more strict typing on String types", :jvariable)
+function jvalue(mx::MxArray, ty::Type{AbstractString})
```
Comment (yuyichao) [inline src/MATLAB.jl]: And this should stay as `jvariable` to avoid printing two depwarns.

**Comment by musm** [comment]
Thanks, fixed.

**Comment by musm** [comment]
merge? @ChrisRackauckas

**Pull request merged**